docs/src/main/sphinx/release/release-353.md

```diff
@@ -2,15 +2,19 @@
 
 ## General
 
+* Add {doc}`/connector/clickhouse`. ({issue}`4500`)
 * Fix aliasing of columns in common table expressions. ({issue}`6839`)
 * Fix potential incorrect query results for queries containing multiple `<` predicates. ({issue}`6896`)
 * Always show `SECURITY` clause in `SHOW CREATE VIEW`. ({issue}`6720`)
 * Fix reporting column references for aliased tables. ({issue}`6972`)
+* Fix potential compiler failure when constructing an array with more than 128 elements. ({issue}`7014`)
```

Comment:
@dain why "potential"?
is it because it depends on constants vs columns?

Reply:
yes exactly